packages/backend/server/tests/copilot-provider.spec.ts

```diff
@@ -111,6 +117,7 @@ test.serial.before(async t => {
 
   registerCopilotProvider(OpenAIProvider);
   registerCopilotProvider(FalProvider);
+  registerCopilotProvider(PerplexityProvider);
```

Comment:
missing a test for perplexity

Reply:
Thanks! I have added the `Search With AFFiNE AI` prompt test in this file. More tests for `PerplexityProvider` are in perplexity.spec.ts.